### ISSUE TITLE
Plotting and animation

### ISSUE THREAD

**Issue by tobanw** [kentwait/nxsim#1]
Are there any plans of porting over the plotting and animation code from ComplexNetworkSim? I'm willing to contribute.

**Comment by kentwait** [comment]
Yes there is, but at the moment I'm cleaning up the code and making some aspects more Pythonic. Once the code base is stable then I can start working on additional features. 

Though if you'd like to contribute porting the plotting code, then that would be great and very much appreciated.
